{"content": {"suites": {"0": {"status": {"total_pass": 3, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 3, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "               ValueError: User with ID 999 not found\n", "test_name": "test_consistency", "rerun": "0"}, "1": {"status": "FAIL", "message": "               KeyError: 'user_id'\n", "test_name": "test_atomicity_commit", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_durability", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_atomicity_rollback", "rerun": "0"}, "4": {"status": "FAIL", "message": "       AssertionError: assert 'Post 1' == 'Updated Post 1'\n         \n         - Updated Post 1\n         + Post 1\n", "test_name": "test_isolation", "rerun": "0"}, "5": {"status": "PASS", "message": "", "test_name": "test_concurrent_updates", "rerun": "0"}}, "suite_name": ".tests/test_acid.py"}}}, "date": "April 23, 2025", "start_time": 1745420281.9563828, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "3", "fail": "3", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "6"}